docs/tactics.md

```diff
@@ -393,6 +393,28 @@ However, it will work, producing the identity function, if one replaces have by
 * `exactI`: `resetI` followed by `exact`. Like `exact`, but uses all
   variables in the context for typeclass inference.
 
+## hint
+
+`hint` lists possible tactics which will make progress (that is, not fail) against the current goal.
+
+```lean
+example {P Q : Prop} (p : P) (h : P → Q) : Q :=
+begin
+  hints,
+  /- the following tactics make progress:
+     ----
+     solve_by_elim
+     finish
+     tauto
+  -/
+  solve_by_elim,
+end
+```
+
+You can add a tactic to the list that `hint` tries by either using
+1. `attribute [hint] my_tactic`, if `my_tactic` is already of type `tactic unit` or `tactic string`, or
```

Comment:
If you use the attribute directly on something of type `tactic unit`, what string does it report on success?

Reply:
Fixed up the comment.